site/src/testHelpers/entities.ts

```diff
@@ -51,6 +51,10 @@ export function assignableRole(role: TypesGen.Role, assignable: boolean): TypesG
   }
 }
 
+export const MockMemberPermissions = {
```

Comment:
Might be helpful to convey permissions in name i.e. `MockMemberWithoutAuditLogPermissions`

Reply:
I added comments, does that help?

Reply:
Yes, it does! Thank you!